games-emulation/dosbox/dosbox-0.74-r1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
The games eclass doesn't support eapi6:
austin@austin2:~/src/gentoo/games-emulation/dosbox$ ebuild dosbox-0.74-r1.ebuild clean install
Appending /home/austin/src/gentoo to PORTDIR_OVERLAY...
 * ERROR: games-emulation/dosbox-0.74-r1::gentoo failed (depend phase):
 *   base.eclass is banned in EAPI 6
 * 
 * Call stack:
 *               ebuild.sh, line 624:  Called source '/home/austin/src/gentoo/games-emulation/dosbox/dosbox-0.74-r1.ebuild'
 *   dosbox-0.74-r1.ebuild, line   6:  Called inherit 'eutils' 'games'
 *               ebuild.sh, line 320:  Called __qa_source '/home/austin/src/gentoo/eclass/games.eclass'
 *               ebuild.sh, line 120:  Called source '/home/austin/src/gentoo/eclass/games.eclass'
 *            games.eclass, line  32:  Called inherit 'base' 'multilib' 'toolchain-funcs' 'eutils' 'user'
 *               ebuild.sh, line 320:  Called __qa_source '/home/austin/src/gentoo/eclass/base.eclass'
 *               ebuild.sh, line 120:  Called source '/home/austin/src/gentoo/eclass/base.eclass'
 *             base.eclass, line  37:  Called die
 * The specific snippet of code:
 *   	6) die "${ECLASS}.eclass is banned in EAPI ${EAPI}";;
 * 
 * If you need support, post the output of `emerge --info '=games-emulation/dosbox-0.74-r1::gentoo'`,
 * the complete build log and the output of `emerge -pqv '=games-emulation/dosbox-0.74-r1::gentoo'`.
 * The ebuild environment file is located at '/var/tmp/portage/games-emulation/dosbox-0.74-r1/temp/environment'.
 * Working directory: '/usr/lib64/python3.4/site-packages'
 * S: '/var/tmp/portage/games-emulation/dosbox-0.74-r1/work/dosbox-0.74'